pgmpy/models/DynamicBayesianNetwork.py

```diff
@@ -542,3 +542,62 @@ def moralize(self):
                 self.get_parents(node), 2))
 
         return moral_graph
+        
+    def copy(self):
+        """
+        Returns the copy of DynamicBayesianNetwork object
+        
+        Examples
+        --------
+        >>> from pgmpy.models import DynamicBayesianNetwork as DBN
+        >>> from pgmpy.factors import TabularCPD
+        >>> student = DBN()
+        >>> student.add_nodes_from(['D', 'G', 'I', 'S', 'L'])
+        >>> student.add_edges_from([(('D', 0),('G', 0)),(('I', 0),('G', 0)),(('D', 0),('D', 1)),(('I', 0),('I', 1))])
+        >>> grade_cpd = TabularCPD(('G', 0), 3, [[0.3, 0.05, 0.9, 0.5],
+        ...                                      [0.4, 0.25, 0.8, 0.03],
+        ...                                      [0.3, 0.7, 0.02, 0.2]],
+        ...                        evidence=[('I', 0),('D', 0)],
+        ...                        evidence_card=[2, 2])
+        >>> d_i_cpd = TabularCPD(('D', 1), 2, [[0.6, 0.3],
+        ...                                    [0.4, 0.7]],
+        ...                      evidence=[('D', 0)],
+        ...                      evidence_card=2)
+        >>> diff_cpd = TabularCPD(('D', 0), 2, [[0.6, 0.4]])
+        >>> intel_cpd = TabularCPD(('I',0), 2, [[0.7, 0.3]])
+        >>> i_i_cpd = TabularCPD(('I', 1), 2, [[0.5, 0.4],
+        ...                                    [0.5, 0.6]],
+        ...                      evidence=[('I', 0)],
+        ...                      evidence_card=2)
+        >>> student.add_cpds(grade_cpd, d_i_cpd, diff_cpd, intel_cpd, i_i_cpd)
+        >>> student.initialize_initial_state()
+        >>> student_copy = student.copy()
+        >>>
+        >>> student_copy.nodes()
```

Comment:
@pdoongarwal Instead of printing the outputs just show that, value of parameters are similar in both copy and original .